### PR TITLE
Highlight deux

### PR DESCRIPTION
**Pull request by dlasky**
better separation of concerns for highlighting and safer output

### CODE REVIEW

**Comment by anthonykoerber** [comment]
👍

**Comment by shuwen** [comment]
lgtm

File: src/strand-dropdown/strand-dropdown.html [hunk 1]
```diff
@@ -51,13 +52,18 @@
 						id="itemRecycler"
 						data="{{data}}"
 						hidden$="{{!_hideInsertionPoints(data)}}"
+						scope="{{scope}}"
```
Comment (jcmoore) [inline src/strand-dropdown/strand-dropdown.html]: Consider changing this to `scope="{{ref}}" and getting rid of the `scope` property  on <strand-dropdown/>

**Comment by jcmoore** [comment]
+1

**Pull request merged**